### PR TITLE
Modify renaming process of Account

### PR DESCRIPTION
**Pull request by shreyashah115**
- Removed Allow Rename from Account master
- Account name and number could only be renamed by clicking on `Update Account Number` button.

account number

### CODE REVIEW

**Comment by rohitwaghchaure** [comment]
Hi @shreyashah115 

- If account number is on the form then set the account number as default in the popup.
- Rename update account number to Update Account Number / Account Name

**Comment by shreyashah115** [comment]
@rohitwaghchaure Done.

File: erpnext/accounts/doctype/account/account.py [hunk 1]
```diff
@@ -188,7 +188,7 @@ def before_rename(self, old, new, merge=False):
 
 		return new_account
 
-	def after_rename(self, old, new, merge=False):
+	def after_not(self, old, new, merge=False):
```
Comment (nabinhait) [inline erpnext/accounts/doctype/account/account.py]: `after_not` ?

**Review by nabinhait — CHANGES_REQUESTED** [review]
- Still need to fix the original issue of splitting in `after_rename` method.
- Add some tests around renaming account

**Review by nabinhait — APPROVED** [review]

**Pull request merged**